### PR TITLE
fix(ui): remove titlecase when displaying environment names

### PR DESCRIPTION
**Pull request by scttcper**
Environments are case sensitive, applying titlecase creates confusion around which environment is being changed/selected.

When hiding or showing an environment, the success toast will now show the environment with the case sensitive environment name.

Closes https://getsentry.atlassian.net/browse/ISSUE-794

### CODE REVIEW

File: src/sentry/static/sentry/app/stores/organizationEnvironmentsStore.jsx [hunk 1]
```diff
@@ -27,7 +26,7 @@ const OrganizationEnvironmentsStore = Reflux.createStore({
       id: item.id,
       name: item.name,
       get displayName() {
-        return toTitleCase(item.name) || DEFAULT_EMPTY_ENV_NAME;
+        return item.name || DEFAULT_EMPTY_ENV_NAME;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/stores/organizationEnvironmentsStore.jsx:29]: Should this call `getDisplayName`?

**Comment by iProgramStuff** [comment]
Looks good to me. I wonder if it's worth adding a test to make sure we aren't just working with lower case names (like "STAGING" or "StAgIng".

**Review by iProgramStuff — APPROVED** [review]
Looks good to me. I wonder if it's worth adding a test to make sure we aren't just working with lower case names (like "STAGING" or "StAgIng").

File: src/sentry/static/sentry/app/stores/organizationEnvironmentsStore.jsx (hunk 1 above)
Comment (scttcper) [reply src/sentry/static/sentry/app/stores/organizationEnvironmentsStore.jsx:29]: @billyvg done 👍

**Review by billyvg — APPROVED** [review]

**Pull request merged**